<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sistema de Reclamações</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Descreva seu problema:</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="txt_reclamacao">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>521</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_limpar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_enviar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>420</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Reclamações enviadas:</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="lt_top_vert">
     <item>
      <layout class="QHBoxLayout" name="lt_hrt_1">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Título:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_titulo"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lt_hrt2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>E-mail:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="txt_email"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPlainTextEdit" name="lbl_reclamacoes">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>291</width>
      <height>371</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_resultado">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>879</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="menu_add"/>
    <addaction name="menu_sair"/>
   </widget>
   <widget class="QMenu" name="menuSobre">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="menu_sobre"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuSobre"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_add">
   <property name="text">
    <string>Salvar</string>
   </property>
  </action>
  <action name="actionListar">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
  <action name="menu_sair">
   <property name="text">
    <string>Sair</string>
   </property>
  </action>
  <action name="menu_sobre">
   <property name="text">
    <string>Sobre</string>
   </property>
  </action>
  <action name="actionListaR">
   <property name="text">
    <string>Listar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
